schema/defs.yaml

```diff
@@ -4,6 +4,11 @@ title: Overture Maps Feature Schema Common Definitions
 description: Common schema definitions shared by all themes
 "$defs":
   propertyDefinitions:
+    entityId:
+      description: >-
+        Optional stable identifier for the feature. 128 bits represented as 32 hex characters.
+      type: string
+      pattern: ^[0-9a-f]{32}$
```

Comment:
Advanced validation thought: we could *probably* figure out a way to assert in validation that `id` != `entityId`.

For later.